cropper/src/main/java/com/canhub/cropper/CropImage.kt

```diff
@@ -177,6 +178,7 @@ object CropImage {
      * @param includeDocuments if to include KitKat documents activity containing all sources
      * @param includeCamera    if to include camera intents
      */
+    @JvmStatic
     fun getPickImageChooserIntent(
```

Comment:
It is possible we add this method usage on the Java sample code? Just wanna be sure no one change it and break in the future =)

Reply:
Done.  Java sample matches Kotlin sample